assets/externalized/strings/shipping-destinations-rus.json

```diff
@@ -0,0 +1,19 @@
+[
+	"Остин",		//Austin, Texas, USA
+	"Багдад",		//Baghdad, Iraq (Suddam Hussein's home)
+	"Драссен",		//The main place in JA2 that you can receive items.  The other towns are dummy names...
+	"Гонконг",		//Hong Kong, Hong Kong
+	"Бейрут",		//Beirut, Lebanon	(Middle East)
+	"Лондон",		//London, England
+	"Лос Анджелес",	//Los Angeles, California, USA (SW corner of USA)
+	"Медуна",		//Meduna -- the other airport in JA2 that you can receive items.
+	"Метавира",		//The island of Metavira was the fictional location used by JA1
+	"Майами",		//Miami, Florida, USA (SE corner of USA)
+	"Москва",		//Moscow, USSR
+	"Нью-Йорк",		//New York, New York, USA
+	"Оттава",		//Ottawa, Ontario, Canada -- where JA2 was made!
+	"Париж",		//Paris, France
+	"Триполи",		//Tripoli, Libya (eastern Mediterranean)
+	"Токио",		//Tokyo, Japan
+	"Ванкувер"		//Vancouver, British Columbia, Canada (west coast near US border)
+]
```

Comment:
Are these used anywhere else? I expected them to be removed from the c++ list, hence the question.

Reply:
These are used to populate the selection lists in Bobby Ray or Florist laptop screens.

Reply:
Missed the removal in the second commit, nevermind.